server/src/database/schema.prisma

```diff
@@ -554,6 +556,46 @@ model Attachment {
   @@map("attachments")
 }
 
+model View {
+  /// @Validator.IsString()
+  /// @Validator.IsOptional()
+  id String @id @default(uuid())
+
+  name       String
+  objectName String
+  fields     ViewField[]
+  sorts      ViewSort[]
+
+  /// @TypeGraphQL.omit(input: true, output: true)
+  workspace   Workspace @relation(fields: [workspaceId], references: [id])
+  /// @TypeGraphQL.omit(input: true, output: true)
+  workspaceId String
+
+  @@unique([objectName, name])
+  @@map("views")
+}
+
+enum ViewSortDirection {
+  ASC
+  DESC
+}
+
+model ViewSort {
+  direction ViewSortDirection
+  key       String
```

Comment:
I think we should add a sortName (similar to fieldName)
We will need to add metadata anyway to handle cases like in people-sorts > fullName

export const availableSorts = [
  {
    key: 'fullname',
    label: 'People',
    icon: <IconUser size={16} />,

    orderByTemplates: [
      (order: Order_By) => ({
        firstName: order,
      }),
      (order: Order_By) => ({
        lastName: order,
      }),
    ],
  },

Reply:
Done: I added a `name` property in the `ViewSort` model.